src/test/java/uk/gov/digital/ho/hocs/info/api/UserIntegrationTest.java

```diff
@@ -48,41 +41,43 @@ public class UserIntegrationTest {
     @Autowired
     ObjectMapper mapper;
 
-    @Before
+    @BeforeEach
```

Comment:
Nope - I carbon copied most of the behaviour from these tests when converting to JUnit5. 

As this is merged, I've noted it for the next PR I do as it can be done atomically.